components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -160,6 +163,7 @@ void BraveReferralsService::Start() {
 }
 
 void BraveReferralsService::Stop() {
+  installation_timer_.reset();
```

Comment:
I would call this `initialization_timer`, since aren't we tracking the initialization of the referral here (vs. the installation of the browser)?

Reply:
Sure. On our side we refer to these as installations, but I'll stick with the convention here.